### PR TITLE
Add OpenGraph meta tags

### PR DESCRIPTION
**Pull request by prtksxna**
Bug: [T177812](https://phabricator.wikimedia.org/T177812)

### CODE REVIEW

**Comment by prtksxna** [comment]
Adding in just one file for now to collect feedback. Will add to the rest if we think we want to go ahead with this.

**Comment by prtksxna** [comment]
Also note, I am adding an `og:image` here even though there is no image on the page. The one that I have chosen here is for general branding.

File: index.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
 	<link rel="stylesheet" href="css/build/wmui-style-guide.min.css">
 	<title>Wikimedia Design Style Guide (WikimediaUI) – provided by Wikimedia Foundation Design Team</title>
+
```
Comment (Volker-E) [inline index.html]: Let's put them after viewport meta.

File: index.html [hunk 2]
```diff
@@ -9,6 +9,12 @@
...
+	<meta property="og:title" content="Wikimedia Design Style Guide (WikimediaUI)" />
```
Comment (Volker-E) [inline index.html]: No self-closing tag notation needed, we're in HTML5 land and that's [unnecessary syntactical sugar](https://stackoverflow.com/a/3558200/1696030).

File: index.html [hunk 3]
```diff
@@ -9,6 +9,12 @@
...
+	<meta property="og:type" content="website" />
+	<meta property="og:url" content="http://design.wikimedia.org/styleguide" />
```
Comment (Volker-E) [inline index.html]: HTTPS missing

**Comment by prtksxna** [comment]
Updated just one file again :)

**Comment by prtksxna** [comment]
@Volker-E Does this look ok to replicate in other files?

**Comment by Volker-E** [comment]
@prtksxna I think that's good to start with OpenGraph vocabulary, yes.

**Comment by Ladsgroup** [comment]
https://design.wikimedia.org/styleguide/img/design-principles/content-first.png 404s for me Are you adding the image in other PRs?

**Comment by prtksxna** [comment]
@Ladsgroup Thanks, even the link to the style guide is incorrect 😮(style-guide, not styleguide). Will fix.

**Pull request merged**